src/main/resources/db/migration/V93__offentlig_transport_vilkår.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE vilkar
+    ADD COLUMN offentlig_transport_reisedager_per_uke INT;
+ALTER TABLE vilkar
+    ADD COLUMN offentlig_transport_pris_enkelbillett INT;
+ALTER TABLE vilkar
+    ADD COLUMN offentlig_transport_pris_trettidagersbillett INT;
```

Comment:
Man kan legge til flere kolonner i samme `ALTER TABLE: ` spørring. Begge funker, men tipper det er noen ytelses fordeler ved en `ALTER TABLE` 

```suggestion
ALTER TABLE vilkar
    ADD COLUMN offentlig_transport_reisedager_per_uke INT,
    ADD COLUMN offentlig_transport_pris_enkelbillett INT,
    ADD COLUMN offentlig_transport_pris_trettidagersbillett INT;
```